### PR TITLE
Add intelicense support for dynamic injected methods on Config

### PR DESCRIPTION
**Pull request by joaoandre-avaiga**
In this PR we add a `.pyi` file which is used for IDEs to have better support for auto completion. It is also add a `MANIFEST.in ` file which is usef for setup.py to provide extra files when installing a library. For now the intelicense file is written by hand, and commited to Github. In the near future we should look into a way to generate this file automatically when packaging the library.

### CODE REVIEW

File: src/taipy/config/config.pyi [hunk 1]
```diff
@@ -0,0 +1,560 @@
...
+                the data node configuration attached to the comparator. See
+                `(taipy.)compare_scenarios()^` more more details.
+            **properties (Dict[str, Any]): A keyworded variable length list of additional
+                arguments.
+        Returns:
+            `ScenarioConfig^`: The new scenario configuration.
+        """
+
+    @classmethod
+    def configure_scenario_from_tasks(
```
Comment (trgiangdo) [inline src/taipy/config/config.pyi]:
```suggestion
    def configure_default_scenario(
```
I think this should be `configure_default_scenario()`

File: src/taipy/config/config.pyi [hunk 2]
```diff
@@ -0,0 +1,560 @@
...
+                configuration if there is a single input data node, or None if there are none.
+            output (Optional[Union[DataNodeConfig^, List[DataNodeConfig^]]]): The list of the
+                function output data node configurations. This can be a unique data node
+                configuration if there is a single output data node, or None if there are none.
+            **properties (Dict[str, Any]): A keyworded variable length list of additional
+                arguments.
+        Returns:
+            `TaskConfig^`: The new task configuration.
+        """
+
```
Comment (trgiangdo) [inline src/taipy/config/config.pyi:596]: We are missing `configure_default_task()` here

File: src/taipy/config/config.pyi [hunk 3]
```diff
@@ -0,0 +1,560 @@
+class Config:
```
Comment (trgiangdo) [inline src/taipy/config/config.pyi:12]: Do we need to add the license at the start of this file?

**Review by trgiangdo — COMMENTED** [review]
I think adding the `config.pyi` file by hand can lead to some mistakes, and it is kinda hard to maintain as well.

If we can not add a script to generate this when setup the library, I propose to write a test to check for the docstring when we run `_inject_section()`, so that we remember to update it when test fails.

File: MANIFEST.in [hunk 4]
```diff
@@ -0,0 +1 @@
+include src/taipy/config/*.pyi
```
Comment (jrobinAV) [inline MANIFEST.in]: Pleas add a new line at the end of the file.

**Review by jrobinAV — DISMISSED** [review]

**Review by jrobinAV — APPROVED** [review]

**Review by jrobinAV — APPROVED** [review]

**Review by trgiangdo — APPROVED** [review]

**Pull request merged**